star-ui/src/components/MappingPanel.tsx

```
@@ -0,0 +1,326 @@
+        <div style={SUB_PANEL}>
+          <div style={SUB_HEADER}>Accumulated Map</div>
+          <canvas
+            ref={mapCanvasRef}
+            width={MAP_SIZE_PX}
+            height={MAP_SIZE_PX}
+            style={CANVAS_STYLE}
+          />
+        </div>
+      </div>
+      <div style={FOOTER_STYLE}>
+        <button style={RESET_BTN_STYLE} onClick={handleReset}>
```

Biome (2.4.4)

[error] Provide an explicit type prop for the button element.

(lint/a11y/useButtonType)